.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+## Your Environment
+Thank you for taking the time to report an issue.
+
+To more efficiently resolve this issue, we'd like to know some basic information about your system and setup.
+
+1) Your OS (as much info as possible)
+* ie: Windows 10, macOS High Sierra version 10.13(17A365), Ubuntu 18.04 LTS, etc)
```

Comment:
Would you be willing to rephrase this? I don't think we need to give examples of an operating system, let's just not use "OS" and instead say "operating system".